File: 8-kyu/even-or-odd..js
// https://www.codewars.com/kata/even-or-odd/train/javascript

function even_or_odd(number) {
  if (number % 2 === 0) {
    return 'Even'
  } else {
    return 'Odd'
  }
}

function even_or_odd(number) {
  return number % 2 === 0 ? 'Even' : 'Odd'
}

const even_or_odd = number => (number % 2 === 0) ? 'Even' : 'Odd'

console.log(even_or_odd(2), "Even")
console.log(even_or_odd(0), "Even")
console.log(even_or_odd(7), "Odd")
console.log(even_or_odd(1), "Odd")

File: rust/8-kyu/september/even-or-odd.rs
// https://www.codewars.com/kata/53da3dbb4a5168369a0000fe/solutions/rust

fn even_or_odd(number: i32) -> &'static str {
    if number % 2 == 0 { "Even" } else { "Odd" }
}

fn even_or_odd(number: i32) -> &'static str {
    match number % 2 {
        0 => "Even",
        _ => "Odd",
    }
}
